backend/src/payloads/AppUser.ts

```diff
@@ -261,3 +261,49 @@ export class AppUserInterviewAvailability {
   @IsDateString()
   end: string;
 }
+
+export class InducteePostRequest {
+  @IsEmail()
+  readonly email: string;
+
+  @IsString()
+  readonly firstName: string;
+
+  @IsString()
+  readonly lastName: string;
+
+  @IsString()
+  readonly major: string;
+
+  @IsString()
+  readonly graduationYear: string;
+
+  @IsString()
+  readonly password: string;
```

Comment:
I don't see you using password for inducteePostRequest so we can remove this I think.